banyand/liaison/grpc/property.go

```diff
@@ -405,11 +411,62 @@ func (ps *propertyServer) Query(ctx context.Context, req *propertyv1.QueryReques
 	return &propertyv1.QueryResponse{Properties: properties, Trace: trace}, nil
 }
 
+func (ps *propertyServer) repairPropertyIfNeed(ctx context.Context, entity string, p *propertyWithCount, groups map[string]*commonv1.Group) error {
```

Comment:
The `repairPropertyIfNeed` function should consider limiting the number of concurrent repair tasks to avoid overwhelming the data nodes with excessive repair traffic. I suggest introducing a limit-size async-queue for repair tasks. 

When adding a new repair task to the queue, please also check for duplicates by comparing the entity and ModRevision to prevent redundant repairs for the same property version. 

This would help ensure the system remains stable under heavy load and avoids unnecessary duplicate repair operations.